### PR TITLE
Improve Book Store exercise description

### PR DESCRIPTION
**Pull request by safwansamsudeen**
@SleeplessByte, this is the PR in response to [this issue](https://forum.exercism.org/t/improve-instructions-of-book-store-exercise/4767/24) on the forum. Thanks!

### CODE REVIEW

**Review by IsaacG — APPROVED** [review]

**Review by SleeplessByte — APPROVED** [review]

File: exercises/book-store/description.md [hunk 1]
```diff
@@ -26,36 +26,36 @@ For example, how much does this basket of books cost?
...
-- 4 × (8 - 1.60) = 4 × 6.40 = $25.60
-- +4 × (8 - 1.60) = 4 × 6.40 = $25.60
+- 4 × (100% - 20%) * $8 = 4 × $6.40 = $25.60, plus
+- 4 × (100% - 20%) * $8 = 4 × $6.40 = $25.60
 
-For a total of $51.20
+equals $51.20
 
-And $51.20 is the price with the biggest discount.
+$51.20 is the price with the biggest discount, so two groups of four is the right answer.
```
Comment (MatthijsBlom) [inline exercises/book-store/description.md:61]: This is wrong: the requested output is a price, not an arrangement.

File: exercises/book-store/description.md [hunk 2]
```diff
@@ -26,36 +26,36 @@ For example, how much does this basket of books cost?
...
 
 Resulting in:
 
-- 5 × (8 - 2.00) = 5 × 6.00 = $30.00
-- +3 × (8 - 0.80) = 3 × 7.20 = $21.60
+- 5 × (100% - 25%) * $8 = 5 × $6.00 = $30.00, plus
+- 3 × (100% - 10%) * $8 = 3 × $7.20 = $21.60
 
-For a total of $51.60
+equals $51.60
```
Comment (kytrinyx) [inline exercises/book-store/description.md:42]:
```suggestion
Which equals $51.60.
```

File: exercises/book-store/description.md [hunk 3]
```diff
@@ -26,36 +26,36 @@ For example, how much does this basket of books cost?
...
 
 Resulting in:
 
-- 4 × (8 - 1.60) = 4 × 6.40 = $25.60
-- +4 × (8 - 1.60) = 4 × 6.40 = $25.60
+- 4 × (100% - 20%) * $8 = 4 × $6.40 = $25.60, plus
+- 4 × (100% - 20%) * $8 = 4 × $6.40 = $25.60
 
-For a total of $51.20
+equals $51.20
```
Comment (kytrinyx) [inline exercises/book-store/description.md:59]:
```suggestion
Which equals $51.20.
```

File: exercises/book-store/description.md (hunk 1 above)
Comment (kytrinyx) [reply exercises/book-store/description.md:61]: @safwansamsudeen for context, please see the canonical data file. The value that this is referring to is "expected", which is the price in cents. https://github.com/exercism/problem-specifications/blob/main/exercises/book-store/canonical-data.json

Comment (safwansamsudeen) [reply exercises/book-store/description.md]: Corrected, my bad.

File: exercises/book-store/description.md (hunk 3 above)
Comment (safwansamsudeen) [reply exercises/book-store/description.md]: Changed!

**Review by IsaacG — APPROVED** [review]

**Review by MatthijsBlom — APPROVED** [review]

**Review by kytrinyx — APPROVED** [review]

**Comment by safwansamsudeen** [comment]
Hey, what's the next step on this PR?

**Pull request merged**